src/content/topics/sdk/features/flag-changes.mdx

```diff
@@ -456,7 +456,7 @@ To subscribe to all feature flag changes, register listeners for change events f
 <CSTab label="JavaScript">
 
 ```js
-client.on('change', (settings) => {
+client.on('update', (settings) => {
```

Comment:
The js sdk does use the 'change' event and not 'update', different from electron so please rollback these changes. Electron uses 'update'.

Reply:
Updated